site/search/SearchUtils.tsx

```diff
@@ -121,12 +121,10 @@ export const getEntityQueryStr = (
 ) => {
     if (!entities?.length) return existingQueryStr
     else {
-        return setSelectedEntityNamesParam(
+        return Url.fromQueryStr(existingQueryStr).updateQueryParams({
             // If we have any entities pre-selected, we want to show the chart tab
-            Url.fromQueryStr(existingQueryStr).updateQueryParams({
-                tab: "chart",
-            }),
-            entities
-        ).queryStr
+            tab: "chart",
+            country: generateSelectedEntityNamesParam(entities),
+        } satisfies GrapherQueryParams).queryStr
```

Comment:
I had to read up on `satisfies` (again!). Beautiful to see it in action :)

Reply:
It's also my first use of it 🎉 

In this case, it's just a way to ensure that the object actually matches that type, and to do that inline. The alternative would be to use `as`, but that just blindly casts it to that type and only checks the rough shape, not that it actually only specifies known properties.

So, yeah, a nice and useful addition to TS :)